tools/ci_connector_ops/ci_connector_ops/pipelines/utils.py

```diff
@@ -246,21 +276,18 @@ def get_modified_connectors(modified_files: Set[Union[str, Path]]) -> dict:
     Or to tests all jdbc connectors when a change is made to source-jdbc or base-java.
     We'll consider extending the dependency resolution to Python connectors once we confirm that it's needed and feasible in term of scale.
     """
+    all_connector_dependencies = [
+        (connector, connector.get_local_dependencies_paths())
```

Comment:
I think the conflict you face is because I renamed `get_local_dependencies_paths` to `get_local_dependency_paths`.